DashAI/back/explainability/local_explainer.py

```diff
@@ -0,0 +1,12 @@
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+
+
+class LocalExplainer:
+    def __init__(self) -> None:
+        pass
+
+    def fit(self, x: DashAIDataset):
+        return self
+
+    def explain_instance(self, x: DashAIDataset):
```

Comment:
Perhaps here for semantics, would it not be better to call x as sample (perhaps to distinguish it from the x, which is commonly used as training dataset)?